frontend/src/bundles/common/components/form-control/form-control.tsx

```diff
@@ -0,0 +1,46 @@
+import { type SxProps, type Theme } from '@mui/material';
+import { FormControl as MUIFormControl } from '@mui/material';
+
+const InputVariantTypes = {
+    FILLED: 'filled',
+    OUTLINED: 'outlined',
+    STANDART: 'standard',
+} as const;
+
+type FormControlVariants =
+    (typeof InputVariantTypes)[keyof typeof InputVariantTypes];
```

Comment:
In this case you need to use ValueOf types to set type for ex: ValueOf<typeof InputVariantTypes>